homework/style.css

```diff
@@ -1,3 +1,165 @@
 .alert-error {
-    color: red;
-}
\ No newline at end of file
+  color: red;
+  text-align: center;
+  font-weight: bold;
+  font-size: 25px;
+  height: 2rem;
+  padding-top: 1rem;
+  margin-top: 50px;
+}
+
+body {
+  width: 768px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 0px;
+  background: #f8f8f8;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell,
+    'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+#main {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+}
+
+#header {
+  display: flex;
+  flex-direction: row;
+  background: #3f51b5;
+  color: white;
+  align-items: center;
+}
+
+.header {
+  margin-top: 17px;
+  margin-left: 20px;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+#repo-select {
+  margin-left: 15px;
+  margin-top: 20px;
+  margin-bottom: 15px;
+  color: rgb(28, 28, 28);
+  border-width: 1.8px;
+  height: 30px;
+  width: 250px;
+  border-radius: 5px;
+  font-size: 14px;
+}
+
+table {
+  table-layout: fixed;
+  color: rgb(0, 0, 0, 81%);
+}
+
+.repo-container,
+.contributor-container {
+  background: white;
+  flex: 1;
+  margin-top: 15px;
+}
+
+.repo-container {
+  margin-right: 5px;
+  padding: 16px;
+}
+
+.left-cell {
+  font-weight: bold;
+}
+
+.right-cell {
+  margin-left: 8px;
+}
+
+.repo-child {
+  font-size: 16px;
+}
```

Comment:
The table formatting doesn't seem completely right. Specifically, a repo description that spans multiple lines should be left-justified, including the first line:

![image](https://user-images.githubusercontent.com/2788771/60345867-414c5880-99ba-11e9-85ab-76de46821d59.png)